pkgs/servers/fiche/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "fiche";
+  version = "0.9.1";
+
+  src = fetchFromGitHub {
+    owner = "solusipse";
+    repo = "fiche";
+    rev = version;
+    sha256 = "1102r39xw17kip7mjp987jy8na333gw9vxv31f7v8q05cr7d7kfb";
+  };
+
+  installPhase = ''
+    install -m 0755 fiche $out/bin
```

Comment:
```suggestion
    mkdir -p $out/bin
    install -m 0755 fiche $out/bin
```

to avoid:

```
install: cannot create regular file '/nix/store/z026bmn69l1lbjgv09vgikqq67b8bnwr-fiche-0.9.1/bin': No such file or directory
```